doc/obtaining.rst

```diff
@@ -0,0 +1,35 @@
+.. _installing:
+
+Obtaining Alex
+==============
+
+.. highlight:: bash
+
+If you just want to *use* Alex, you can build from a release.
+This should work the same as any other Haskell package.
+
+Alex itself and its examples are intended to work with GHC >= 7.0.
+
+Haskell-specific way
+--------------------
+
+From `Hackage <https://hackage.haskell.org/package/alex>`__ via `Cabal Install <https://www.haskell.org/cabal/>`__::
+
+   $ cabal install alex
+
+From `Stackage <https://www.stackage.org/package/alex>`__ via `Stack <https://haskellstack.org>`__::
+
+   $ stack install --resolver nightly alex
```

Comment:
Does this also work without the `--resolver` flag?

Reply:
No idea, this was taken as is from the website.